### PR TITLE
refactor: Inline and simplify glowing backdrop SVG pattern

### PR DESCRIPTION
**Pull request by damianstasik**
## Description

The SVG pattern used for the glowing backdrop is quite big, fortunately thanks to compression instead of transferring 50kb only around 6kb is sent through the wire. There is still issue of the additional request just to show a pattern which blinks on page load no matter how fast your Internet connection might be. This PR is an experiment to simplify the SVG and inline it into a React component so that we do not make unnecessary requests. 

To simplify the SVG I've tried to run it through SVGO, but there is a lot of separate/unmergable paths and a bit of unnecessary complexity so the outcome was not as good as it could be. I've decided to remake the it in [Figma](https://www.figma.com/file/gGTrU1hsWQZjowWMvdSWkQ/Node.js-Grid-by-DS?type=design&node-id=0%3A1&mode=design&t=Cb5AaxvEVVUdRYy1-1) from scratch, make it pixel-perfect, controllable with local vars to make further adjustments much easier, then I flattened the paths and exported the result.

Old SVG – pre-optimized: 48.28kb, optimized: 11.38k (1.18kb gzipped)
New SVG – pre-optimized: 6.81k, optimized: 4.89k (673b gzipped)

I still have a few ideas how to bring that down even more, but I'll need a bit more time if there's an interest, if no feel free to close this PR.

### Check List

- [ ] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [ ] I have run `npx turbo format` to ensure the code follows the style guide.
- [ ] I have run `npx turbo test` to check if all tests are passing.
- [ ] I have run `npx turbo build` to check if the website builds without errors.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Comment by AugustinMauroy** [comment]
Just something strange the design had change. it's can from you svg manipulation. Try to keep SVG as it

**Comment by AugustinMauroy** [comment]
it's can solve #6520

**Comment by ovflowd** [comment]
> it's can solve #6520

Probably yes, yea.

File: styles/effects.css [hunk 1]
```diff
@@ -10,29 +10,3 @@ h1.special {
...
     md:-tracking-[0.06rem];
 }
-
-div.glowingBackdrop {
-  @apply absolute
-    left-0
-    top-0
-    -z-10
-    size-full
-    bg-[url('/static/images/patterns/hexagon-grid.svg')]
```
Comment (MattIPv4) [inline styles/effects.css:20]: Is this image being used elsewhere, or can it be removed as part of this?

Comment (ovflowd) [reply styles/effects.css:20]: Good point

Comment (bmuenzenmeyer) [reply styles/effects.css:20]: it looks to be used in `Preview`

**Comment by AugustinMauroy** [comment]
Note we have already an component `HexagonGrid` use on OG generation

**Comment by ovflowd** [comment]
Hey @damianstasik any updates to the PR?

**Comment by ovflowd** [comment]
@AugustinMauroy do you mind taking over this PR?

**Review by ovflowd — APPROVED** [review]

**Pull request merged**